packages/web-components/src/components/cbp-action-bar/cbp-action-bar.tsx

```diff
@@ -0,0 +1,27 @@
+import { Component, Host, Prop, h } from '@stencil/core';
+
+@Component({
+  tag: 'cbp-action-bar',
+  styleUrl: 'cbp-action-bar.scss',
+})
+export class CbpActionBar {
+
+  /** variant for type: sticky/inline */
+  @Prop({ reflect: true }) variant: 'inline' | 'sticky'
+
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
+
```

Comment:
You need to bring in the parts that enable the `sx` property. See any other component for an example. This should always be last - move context above it.